presidio-anonymizer/README.MD

```diff
@@ -0,0 +1 @@
+# TODO add the document Omri wrote.
```

Comment:
Not sure this comment is understandable

Reply:
sorry I missed this comment. I already removed it in my next commit. it got here by accident.